html/rename.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Rename</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="/static/css/bootstrap.min.css">
+</head>
+
+<body>
+    {{template "navbar"}}
+
+    <div class="container">
+
+        <div class="container pt-5 pl-0">
+            <h2>Device Rename<br></h2>
+        </div>
+        <hr>
+
+        <form>
+          <div class="form-row">
+            <div class="form-group col-md-6">
+              <label for="new-name">New name</label>
+              <input id="new-name" type="text" class="form-control" placeholder="name">
+            </div>
+          </div>
+
+          <div class="form-row">
+            <div class="form-group col-md-6">
+              <label for="new-group">New Group</label>
+              <input id="new-group" list="group-list" name="new-group" class="form-control" placeholder="group"/>
+              <datalist id="group-list">
+              </datalist>
```

Comment:
Android WebViews don't support `<datalist>` and I'd like to see about having Sidekick use a WebView instead of calling out to an external browser. You could introduce this polyfill: https://github.com/mfranzke/datalist-polyfill to ensure it works.